### PR TITLE
fix: tie shows No Ruling

### LINKED ISSUE #46: Remove the upper left  bar
On some browsers, there is an upper left sliding bar which can't be used. We need to remove it.

### PR DESCRIPTION
**Pull request by satello**
Fixes #46 (should verify), #50, #49

### CODE REVIEW

File: src/components/nav-bar/nav-bar.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
   background: $dark3;
   display: flex;
   height: 70px;
+  overflow-x: hidden;
```
Comment (epiqueras) [inline src/components/nav-bar/nav-bar.scss]: I think the problem lies in `&-tabs`.

File: src/containers/disputes/components/disputes-table/index.js [hunk 2]
```diff
@@ -35,7 +35,17 @@ const columns = [
 ]
 
 const DisputesTable = ({ disputes }) => (
-  <Table columns={columns} loading={disputes.loading} data={disputes.data} />
+  <Table
+    columns={columns}
+    loading={disputes.loading}
+    data={disputes.data}
+    defaultSorted={[
```
Comment (epiqueras) [inline src/containers/disputes/components/disputes-table/index.js]: Is the array of disputes fetched from `0...disputes.length` (oldest...newest)?

Comment (satello) [reply src/containers/disputes/components/disputes-table/index.js]: Yeah it is. So it comes in ordered by disputeId, which isn't necessarily oldest to newest with appeals

File: src/components/nav-bar/nav-bar.scss (hunk 1 above)
Comment (satello) [reply src/components/nav-bar/nav-bar.scss]: 👍

**Pull request merged**